pages/5g-multicast-broadcast-services/ran-aspects.md

```diff
@@ -0,0 +1,54 @@
+has_children: false
+nav_order: 2
+---
+
+{: .warning }
+This documentation is currently **under development and subject to change**. It reflects outcomes elaborated by 5G-MAG members as part of **WI: Multicast Broadcast Services**
+We welcome and encourage contributions from the broader community. If you are interested in becoming a member of the 5G-MAG and actively participating in shaping this work, please contact the [Project Office](https://www.5g-mag.com/contact)
+
+# MBS RAN Aspects
+
+## RAN traffic delivery methods (TS 38.300/331 when complete)
+On receiving an MBS packet from the 5GC via the shared MBS traffic delivery method, the gNodeB can choose between two different RAN delivery methods, as described below.
+
+### Point-to-Multipoint (PTM) RAN delivery method
+The PTM RAN delivery method is used only for MBS packets arriving at the gNodeB via the 5GC shared MBS traffic delivery method. gNodeBs then deliver a single copy of the MBS data packets over the radio interface to multiple UEs. The radio interface may use either delivery mode 1 (multicast) or delivery mode 2 (broadcast).
+The PTM RAN delivery method is always used to deliver the MBS packets of a Broadcast MBS Session.
+The PTM RAN delivery method may be used to deliver the MBS packets of a Multicast MBS Session. In some cases, however, the gNodeB may opt to use the PTP delivery method (see below)
+
+### Point-to-Point (PTP) RAN delivery method
+The PTP RAN delivery method may be used for MBS packets arriving at the gNodeB via either the 5GC shared MBS traffic delivery method or the 5GC individual MBS traffic delivery method. gNodeBs then deliver separate copies of the MBS data packets over the radio interface to individual UEs.
+The PTP RAN delivery method is always used for conventional PDU Sessions carrying packets from a unicast session.
+The PTP RAN delivery method is always used in combination with the 5GC individual MBS traffic delivery method.
+The PTP RAN delivery method may be used in combination with the 5GC shared MBS traffic delivery method – for example in cases where specific UEs require more robust Modulation Coding Schemes (MCS) to achieve reliable reception, or if the number of UEs within a given gNodeB subscribing to the Multicast MBS Session is below a certain threshold. In such cases, the gNodeB replicates MBS packets into UE-specific Radio Resource Blocks.
+The PTP RAN delivery method is not used for broadcast MBS sessions.
+
+## RAN delivery modes
+5G-NR MBS compatible gNodeBs support three different delivery modes at Layer 2: delivery mode 1 (multicast), delivery mode 2 (broadcast), and the unicast delivery mode, where the latter is italicised as it does not reflect a defined 3GPP’s term but is used in this document to describe the ‘default unicast’ delivery mode, absent 5G-NR MBS. 
+
+### Unicast delivery mode
+gNodeBs always use the unicast delivery mode for the point-to-point RAN delivery method. This includes the delivery of:
```

Comment:
Is `unicast delivery mode` a term, which is used in 3GPP specifications? Could it be enough with `point-to-point RAN delivery method` or `point-to-point delivery method`